src/pages/ThorChainLP/queries/hooks/useUserLpData.ts

```diff
@@ -122,6 +122,8 @@ export const useUserLpData = ({
     queryFn: async ({ queryKey }) => {
       const [, , , { assetId }] = queryKey
 
+      console.log('useUserLpData query')
```

Comment:
Is this guy intentionally left in?

Reply:
I'm going to assume not, removed in https://github.com/shapeshift/web/pull/6408/commits/014aeda0fbd94542fc963a461f196cf932ffe75d.